src/libraries/System.Formats.Asn1/src/System.Formats.Asn1.csproj

```diff
@@ -0,0 +1,61 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <UsePackageTargetRuntimeDefaults>true</UsePackageTargetRuntimeDefaults>
+    <IncludeDllSafeSearchPathAttribute>true</IncludeDllSafeSearchPathAttribute>
```

Comment:
I assume this is just to add the DllSafeSearchPath attribute to the assembly right?